### ISSUE TITLE
Bug: when id is 0, unexpected issues occur

### ISSUE THREAD

**Issue by bofeiw** [lourenci/react-kanban#178]
When assigning 0 as id to a lane, i observed  at least these unexpected issues:
- after drag complete, the orders of lanes might change
- after drag complete to another lane, the card might remains at the original lane

For example, modifying line 10 of the demo in readme:

https://codesandbox.io/s/react-kanban-demo-mfulg

```javascript
import React from "react";
import ReactDOM from "react-dom";
import Board from "@lourenci/react-kanban";

import "./styles.css";

const board = {
  lanes: [
    {
      id: 0, ////////////////////////// <= causing issues
      title: "Backlog",
      cards: [
        {
          id: 1,
          title: "Card title 1",
          description: "Card content"
        },
        {
          id: 2,
          title: "Card title 2",
          description: "Card content"
        },
        {
          id: 3,
          title: "Card title 3",
          description: "Card content"
        }
      ]
    },
    {
      id: 2,
      title: "Doing",
      cards: [
        {
          id: 9,
          title: "Card title 9",
          description: "Card content"
        }
      ]
    },
    {
      id: 3,
      title: "Q&A",
      cards: [
        {
          id: 10,
          title: "Card title 10",
          description: "Card content"
        },
        {
          id: 11,
          title: "Card title 11",
          description: "Card content"
        }
      ]
    },
    {
      id: 4,
      title: "Production",
      cards: [
        {
          id: 12,
          title: "Card title 12",
          description: "Card content"
        },
        {
          id: 13,
          title: "Card title 13",
          description: "Card content"
        }
      ]
    }
  ]
};

function App() {
  return (
    <Board
      allowRemoveLane
      allowRenameLane
      allowRemoveCard
      onLaneRemove={console.log}
      onCardRemove={console.log}
      onLaneRename={console.log}
    >
      {board}
    </Board>
  );
}

const rootElement = document.getElementById("root");
ReactDOM.render(<App />, rootElement);
```

**Comment by bofeiw** [comment]
It is not a big issue, but when you use some id generators like https://www.npmjs.com/package/increment-generator , it is extremely annoying because the first number it returns is 0.

It took me quite some time to spot the bug.
Hope it can be resolved, or at least warn users  not to use 0 as id cos it'll save users's time

**Comment by lourenci** [comment]
Hi, @bofeiw. Thanks for the feedback. I'll look at this soon.

**Comment by lourenci** [comment]
Hi, @bofeiw. I think this is not an issue in the most recent version of the lib. Could you update it and verify it? I've tested in the version released today (0.15.0) and it seems ok.

**Comment by bofeiw** [comment]
> Hi, @bofeiw. I think this is not an issue in the most recent version of the lib. Could you update it and verify it? I've tested in the version released today (0.15.0) and it seems ok.

It still does not work with 0.15.0 here: https://codesandbox.io/s/react-kanban-demo-mfulg

Here is what i got:
https://streamable.com/to1vx

**Comment by lourenci** [comment]
@bofeiw, in the 0.15 version, `children` was renamed to `initialBoard` for the uncontrolled board. Can you change that prop and test it again?

**Comment by bofeiw** [comment]
It appears that the issue is solved in 0.15, thanks
